examples/15_scipp_loader.ipynb

```diff
@@ -0,0 +1,340 @@
+{
```

Comment:
Is the notebook important to add or does the `.py` script + README have the same details?

Reply:
@ax3l the  README from my repo is identical to the notebook since I was generating my `README.md`  from it with:
```bash
rm -rf README_files
python -m nbconvert --to notebook --execute README.ipynb --output README.executed.ipynb
python -m nbconvert --to markdown README.executed.ipynb --output README.md - TagRemovePreprocessor.enabled=True --TagRemovePreprocessor.remove_cell_tags remove_cell
rm -f README.executed.ipynb
```
so you don't need both